examples/README.md

```diff
@@ -0,0 +1,40 @@
+## Structure
+
+For the simplest examples without using any helping code (see `framework.rs` here), check out:
+  - `hello ` for printing adapter information
+  - `hello-triangle` for graphics and presentation
+  - `hello-compute` for pure computing
+
+Notably, `capture` example show rendering without a surface/window. It reads back the contents and saves them to a file.
+
+All framework-based examples render to the window.
+
+## Feature matrix
+| Feature                | boids  | cube   | mipmap | msaa-line | shadow | skybox | texture-arrays | water  |
+| ---------------------- | ------ | ------ | ------ | --------- | ------ | ------ | -------------- | ------ |
+| vertex attribute       | :star: | :star: | :star: | :star:    | :star: |        | :star:         | :star: |
```

Comment:
Even though the features themselves are generally not pluralized, I would prefer to see these pluralized as it reads much better in docs.

Reply:
yeah, I unconsciously pluralized a few things anyway :)